### DISCUSSION TITLE
Can't get Google Services Framework Android ID

### DISCUSSION THREAD

**Discussion by PewPewCricket** [waydroid/waydroid#1335] [Q&A]
I am attempting to run the command given on the official site:
ANDROID_RUNTIME_ROOT=/apex/com.android.runtime ANDROID_DATA=/data ANDROID_TZDATA_ROOT=/apex/com.android.tzdata ANDROID_I18N_ROOT=/apex/com.android.i18n sqlite3 /data/data/com.google.android.gsf/databases/gservices.db "select * from main where name = \"android_id\";"

this command just exits and prints nothing to the waydroid shell, I can't find a solution to this issue and would appreciate some help, thank you!

**Comment by FalconSN** [comment]
You need to escape inner double quotes.

**Reply by PewPewCricket** [reply]
This isn't helpful at all

**Reply by FalconSN** [reply]
Some problems are caused by simple mistakes. Take a look at the size of gservices.db. If it's 0 bytes, there was someone with same problem in Telegram, you can search in the chat history hoping they explained what caused it.
